### ISSUE TITLE
Unexpected Magikeyboard layout

### ISSUE THREAD

**Issue by bakedonion** [Kunzisoft/KeePassDX#927]
**Describe the bug**
When switching to the Magikeyboard I get an unexpected keyboard layout for a specific app. Normally I get a layout with the _entry select_ or _entry elements_ buttons and the button to switch back to the previous keyboard:

![image](https://user-images.githubusercontent.com/56303754/111048798-aaedee00-844d-11eb-9918-d3e560513a93.png)

But for this app I receive a different layout and no option to switch back to my previous keyboard:

![image](https://user-images.githubusercontent.com/56303754/111048509-97db1e00-844d-11eb-9827-9cf549597829.png)

Likewise (in case autofill is enabled in the settings), there is no autofill dropdown. In the wiki there is a section about [Autofill disabled](https://github.com/Kunzisoft/KeePassDX/wiki/AutoFill#autofill-disabled), could this be related?

The app in question is [S-pushTAN](https://play.google.com/store/apps/details?id=com.starfinanz.mobile.android.pushtan). So far this is the only case I encountered this.

**To Reproduce**
1. Open the S-pushTAN app
2. Click on the password field
3. Switch to the Maigkeyboard
4. See unexpected layout

**Expected behavior**

I would have expected to see the same layout as seen here:

![image](https://user-images.githubusercontent.com/56303754/111048798-aaedee00-844d-11eb-9918-d3e560513a93.png)

If this is related to the app and requires changes on their part, I would expect to still have the option to switch back to my previous keyboard. Currently there is a need to switch out of the app in question in order to be able to change it again.

**KeePass Database**
 - Created with: KeePassDX
 - Version: 2.9.14

**KeePassDX (please complete the following information):**
 - Version: 2.9.14
 - Build: Free
 - Language: English

**Android (please complete the following information):**
 - Device: Moto G6 Plus
 - Version: 9

**Additional context**
- App in question: [S-pushTAN](https://play.google.com/store/apps/details?id=com.starfinanz.mobile.android.pushtan)

**Comment by J-Jamet** [comment]
I can't even launch the application which crashes at startup, which doesn't bode well for the seriousness of the code if this app in general. So I don't know, I'll look at other devices when I have time.

**Comment by bakedonion** [comment]
Did you try this in an AVD? The app in question is a banking app. While writing this issue, the app prevented me from taking screenshots which is why I tried to accomplish it by running them in an emulated device, where I also ran into crashes.

I am just guessing here, but maybe they are performing some additional device checks or something of that sort? If it was an actual device then I am clueless, sorry.

**Comment by J-Jamet** [comment]
Yes, I tried with an emulator and didn't look any further.

**Comment by J-Jamet** [comment]
OK, I just installed your bank's application on one of my test phones (it starts up this time).

_Magikeyboard :_ 
It seems that this application blocks custom keyboards, I don't know their technique but whether it is the Magikeyboard, Simple Keyboard or the KeePass2Android keyboard, they are blocked.
I set breakpoints, and no KeePassDX keyboard code is executed. So it's blocked before the keyboard is chosen and a default keyboard is displayed.

_Autofill :_
I tested the auto-fill, it works when the google keyboard is activated. (On the "pushTAN-Passwort" field, so the autofill works well, at least for the field I tested)
